lib/find.js

```diff
@@ -138,11 +176,11 @@ var getTimezone = function (originalLat, originalLon) {
     if (!curTzData) {
       // no timezone in this quad, therefore must be timezone at sea
       return getTimezoneAtSea(originalLon)
-    } else if (curTzData === 'f') {
+    } else if (curTzData.pos && curTzData.len) {
```

Comment:
It seems like the check for `.pos` should be a little more explicit to avoid JavaScript truthiness of integers of value `0` and make sure to check for a file that starts at position `0`. See recommendation:

```suggestion
    } else if (curTzData.pos >= 0 && curTzData.len) {
```